tdms/CMakeLists.txt

```
@@ -61,6 +64,25 @@ else()
 endif()
 
 
+# spdlog ----------------------------------------------------------------------
+find_package(spdlog NO_CMAKE_PACKAGE_REGISTRY QUIET)
+if(NOT spdlog_FOUND)
+    include(FetchContent)
+    FetchContent_Declare(spdlog
+            GIT_REPOSITORY https://github.com/gabime/spdlog.git
+            GIT_TAG v1.10.0
+            )
+    FetchContent_MakeAvailable(spdlog)
+endif()
+
+# Set the active logging level
+if("${CMAKE_BUILD_TYPE}" STREQUAL "Test")
```

Was this meant to be
```suggestion
if("${CMAKE_BUILD_TYPE}" STREQUAL "Debug" OR "${CMAKE_BUILD_TYPE}" STREQUAL ""${CMAKE_BUILD_TYPE}" STREQUAL "RelWithDebInfo"")
```
since we aren't using a "Test" [build type](https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html) anywhere?

---

we aren't using "Debug" or "RelWithDebInfo" either 🤷🏼. Don't have any opinions really – happy to go with whatever's common